src/components/Organisms/TodoCard/index.jsx

```diff
@@ -1,11 +1,22 @@
 import styled from "styled-components";
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import AddTaskButton from "../../Atoms/AddTaskButton/index.jsx";
 import Task from "../../Molecules/Task/index.jsx";
 import COLOR from "../../../variables/color.js";
 
 const TodoCard = () => {
   const [taskList, setTaskList] = useState([]);
+  useEffect(() => {
+    const data = localStorage.getItem("data");
+    if (data) {
+      const liftedTaskList = JSON.parse(data);
+      setTaskList(liftedTaskList);
+    }
+  }, []);
+  useEffect(() => {
+    const EditedTaskList = JSON.stringify(taskList);
+    localStorage.setItem("data", EditedTaskList);
+  });
```

Comment:
```suggestion
  useEffect(() => {
    const editedTaskList = JSON.stringify(taskList);
    localStorage.setItem("data", editedTaskList);
  },[taskList]);
```
>  If you omit this argument, your Effect will re-run after every re-render of the component.

引用：https://react.dev/reference/react/useEffect#useeffect
useEffectの依存配列を省くと、コンポーネントが再レンダリングするごとに実行されてしまうので上記のように依存配列を設定するようお願いします！ 

また、 `editedTaskList` はコンポーネントではなく普通のオブジェクトなので、ローワーキャメルケースで書くようにしてください！